kube_controller_manager/hatch.toml

```diff
@@ -5,7 +5,7 @@ base-package-features = [
 ]
 
 [envs.default]
-e2e-env = false
+e2e-env = true
```

Comment:
`true` is the default value with hatch (but `false` is the default value with tox), this is defined [here](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/plugin/hatch/environment_collector.py#L71-L73) so you can remove this section from the hatch file: 

```
[envs.default]
e2e-env = true
```

Reply:
Hmm, I thought I added those because I don't think I would've been able to run the E2E environments without those. Thanks though, I'll triple check I've enabled them and add the missing license information